### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by janczer**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private JPanel getCorePanel(boolean update) {
 							java.awt.Color.black));
 
 			if (latestInfo == null || this.latestInfo.getZapRelease() == null) {
-				// Havnt checked for updatees yet
+				// Havnt checked for updates yet
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java]: Could fix ‘havnt’ too

File: src/org/apache/commons/httpclient/HttpConnection.java [hunk 2]
```diff
@@ -840,7 +840,7 @@ public void setTunnelRequested(boolean tunnelRequested) {
     /**
      * Indicates if the connection is completely transparent from end to end.
      *
-     * @return true if conncetion is not proxied or tunneled through a transparent
+     * @return true if connection is not proxied or tunneled through a transparent
```
Comment (thc202) [inline src/org/apache/commons/httpclient/HttpConnection.java]: Please, revert the changes in these forked classes, they should have just essential/required changes for our use cases.

File: src/org/zaproxy/zap/utils/DisplayUtils.java [hunk 3]
```diff
@@ -75,15 +75,15 @@ public static ImageIcon getScaledIcon(ImageIcon icon) {
 	
 	public static Icon getScaledIcon(Icon icon) {
 		if (! isScaleImages() || icon == null || FontUtils.getScale() == 1 || ! (icon instanceof ImageIcon)) {
-			// dont need to scale (or cant)
+			// don't need to scale (or cant)
```
Comment (kingthorin) [inline src/org/zaproxy/zap/utils/DisplayUtils.java]: Could fix ‘cant’ as well

File: src/org/apache/commons/httpclient/HttpConnection.java (hunk 2 above)
Comment (janczer) [reply src/org/apache/commons/httpclient/HttpConnection.java]: Done

File: src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java (hunk 1 above)
Comment (janczer) [reply src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java]: Fixed

File: src/org/zaproxy/zap/utils/DisplayUtils.java (hunk 3 above)
Comment (janczer) [reply src/org/zaproxy/zap/utils/DisplayUtils.java]: Fixed

File: src/org/parosproxy/paros/extension/history/ProxyListenerLog.java [hunk 4]
```diff
@@ -54,7 +54,7 @@
     private static final Logger log = Logger.getLogger(ProxyListenerLog.class);
 
     // ZAP: Must be the last one of all listeners to be notified, as is the one that saves the HttpMessage 
-	// to the DB and must let other listeners change ase' and testthe HttpMessage before saving it.
+	// to the DB and must let other listeners change ase' and test the HttpMessage before saving it.
```
Comment (thc202) [inline src/org/parosproxy/paros/extension/history/ProxyListenerLog.java]: `ase'` could be removed.

Comment (janczer) [reply src/org/parosproxy/paros/extension/history/ProxyListenerLog.java]: Done

**Comment by thc202** [comment]
Thanks!
